### ISSUE TITLE
ia32 builds are missing since 1.78.1.23130

### ISSUE THREAD

**Issue by al-lac** [VSCodium/vscodium#1519]
**Describe the bug**
32-bit builds are missing since [1.78.1.23130](https://github.com/VSCodium/vscodium/releases/tag/1.78.1.23130)

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
Check the release page.

**Expected behavior**
32-bit builds should be available.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows, macOS, Linux
 - Architecture ia32
 - Version 1.78.1.23130
 - App Manager All
 - Sandboxed no

**Comment by adomasbaliuka** [comment]
The same seems to be true for `reh-linux-armhf`. As a result, using "Open Remote - SSH" is no longer possible when connecting, e.g., to a Raspberry Pi. The extension tries to download

    https://github.com/VSCodium/vscodium/releases/download/1.78.2.23132/vscodium-reh-linux-armhf-1.78.2.23132.tar.gz

but this target is no longer built in the new versions.

The path can be changed in the extension (setting `remote.SSH.serverDownloadUrlTemplate`), but so far I haven't found anything I could put there that would work when using the current vscodium version on the host machine (it says the download is corrupted).

**Comment by jeanp413** [comment]
cc @daiyam

**Comment by jeanp413** [comment]
@daiyam you need to update the linux build agent with upstream changes

**Comment by daiyam** [comment]
@jeanp413 Oh! That could explain the issue on Linux but not on Windows (same error)...

**Comment by daiyam** [comment]
Maybe related to this: https://github.com/microsoft/vscode/blob/4137aa15f55c5ec3777daf6684908f64b439f6aa/build/azure-pipelines/win32/product-build-win32.yml#L93

**Comment by daiyam** [comment]
No luck with the updates build agent for linux.
But I was able to fix the windows build...

**Comment by jeanp413** [comment]
Did you update the linux build to use the new image they use `buster` now

**Comment by daiyam** [comment]
Ya. Yes, `buster` was added for that version. Next version seems to use new `bionic` (disable but still used in their azure build)

**Comment by daiyam** [comment]
`node-gyp` was also failing on `strech-armv7l` but the errors were ignored...
It seems to configuration issue (`gyp info using node@16.20.0 | linux | x64`) even with the env variable `npm_config_arch=armv7l`

**Comment by jeanp413** [comment]
yeah that looks strange but it compiles fine for openvscode-server, though I see openvscode-server is using gyp 9.1.0 and node 16.20.0 :thinking: 
![image](https://github.com/VSCodium/vscodium/assets/25115070/e432ef2a-60ed-4763-ba3a-d5f928c89c7c)

**Comment by jeanp413** [comment]
@daiyam I think it should be `export npm_config_arch="arm"` instead of `armv7l` 
![image](https://github.com/VSCodium/vscodium/assets/25115070/ade7e804-d19a-4689-b980-8dfcf07466f9)

**Comment by daiyam** [comment]
`.nvmrc` in root uses `16.17` while CI use `16.x`...

**Comment by daiyam** [comment]
`export npm_config_arch="arm"` same error

**Comment by daiyam** [comment]
`16.20` same error...

**Comment by jeanp413** [comment]
@daiyam this commit looks odd https://github.com/VSCodium/vscode-linux-build-agent/commit/cb7ca1192e95c83ea8ef1db2d548c70501b22930 :thinking: why is using stretch in the buster image? maybe revert it and try again

**Comment by daiyam** [comment]
Aaaaahhh!  :sob:

Thx

**Comment by daiyam** [comment]
OK the error is shorter (back to the usual one)
I've tried `bionic`, same error

**Comment by jeanp413** [comment]
could you try removing this line https://github.com/VSCodium/vscodium/blob/272f67d95b6cce7545822247791ba1eef861ef0a/prepare_vscode.sh#L68

**Comment by daiyam** [comment]
The 32bits builds are available!

@jeanp413 Thank you!
